login.php

```diff
@@ -0,0 +1,31 @@
+<?php include('includes/header.php');?>
+<div id="page-container">
+
+    <div id="content-wrap">
+
+        <form action="" method="POST">
+
+            <div class="container">
+
+                <h1>Login Form</h1>
+                <p>Please fill in this form to create an account.</p>
+                <hr>
+
+                <label>Email Address:</label><br>
+                <input type="email" name="email" id="email" value="<?php echo $email; ?>">
+
+                <br><label>Password:</label><br>
+                <input type="password" name="password" id="password" >
+
+                <p>By creating an account you agree to our <a href="#" style="color:dodgerblue">Terms & Privacy</a>.</p>
```

Comment:
Please remove all inline styles. If you need a color modifier you can use one of our utility classes but we'll be coming back and making modifications to this code so don't worry if it doesn't look 100% perfect: http://treatstreet.herokuapp.com/styleguide/utilities/